css/css-structure_selectors-and-cascading/css/styles.css

```diff
@@ -49,92 +49,93 @@ nav li {
 /* -------------------------------------------------------- */
 
 /* 1. These styles should apply to the headline */
-FIX_ME {
+h1 {
   outline: 2px dashed red;
   font-size: 2rem;
   margin: 0 2rem 0 0;
   color: #2673bb;
 }
 
 /* 2. These styles should apply to all links */
-FIX_ME {
+a {
   outline: 2px dashed red;
   color: #ff00e6;
 }
 
 /* 3. These styles should apply to all external links (you can use the attribute selector) */
-FIX_ME {
+a[href="https://developer.mozilla.org/de/"]
```

Comment:
This would target all links, that lead to exactly this url. The right attribute should be "target" - just put that between the [ ] without anything else - voila, you targeted all external links. :)